.travis.yml

```diff
@@ -32,3 +37,20 @@ deploy:
   skip_cleanup: true
   on:
     tags: true
+
+after_deploy:
+  - 'git clone https://github.com/Appliscale/rpmbuild.git'
+  - 'cd rpmbuild/SOURCES'
+  - 'rm perun-linux-amd64.tar.gz'
+  - 'wget -P SOURCES https://github.com/Appliscale/perun/releases/download/1.2.0/perun-linux-amd64.tar.gz'
```

Comment:
We want to download binary from tag we are curently releasing, not hardcoded `1.2.0`. You can get it from `TRAVIS_TAG` envirnoment variable.